src/sass/navbar.scss

```
@@ -36,6 +36,14 @@ nav {
         &.show, &.active {
           color: #000;
         }
+
+        .ai-icon {
+          position: relative;
+          top: 0.125rem;
+          background: -webkit-linear-gradient(45deg, rgb(169, 114, 255), rgb(41, 151, 255) 50%, rgb(67, 185, 185));
```

Stylelint (17.5.0)

[error] Unexpected vendor-prefixed value "-webkit-linear-gradient" (value-no-vendor-prefix)

(value-no-vendor-prefix)